### PR TITLE
fix(DB/Creature) Correct order of Shandy Glossgleam dialogue

### PR DESCRIPTION
**Pull request by YggdrasilWotLK**
Makes Shandy say his for a Suitable Disguise in the correct order

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [X] Database (SAI, creatures, etc).

Order of dialogues was mixed up, causing issues such as the quest seeming like it had failed (ie. Sandy said the failure dialogue) despite the fact that there's no scripted failure event in the underlying cpp script, or asking for shirts instead of saying the second half of the scripted intro RP

## Issues Addressed:

- Closes 
https://github.com/azerothcore/azerothcore-wotlk/issues/744

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [X] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

https://www.youtube.com/watch?v=DHmeKq9t5Z0

## Tests Performed:

This PR has been:
- [X] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [X] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. .quest add 24556 (horde) or .quest add 20438 (ally)
2. .go creature 1790
3. initiate the quest RP and see the dialogue mismatch

## Known Issues and TODO List:

- [X] The failure event is missing

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/shandy-fix.sql
Comment (Helias) [inline data/sql/updates/pending_db_world/shandy-fix.sql:4]: out the backtick ` in all columns and fields names (same in the bottom SQL query)

Comment (YggdrasilWotLK) [reply data/sql/updates/pending_db_world/shandy-fix.sql]: Backticked

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
I do not work much on these tables so I ask a second opinion.
About testing, the SQL scripts should be enough simple to not require other tests than the author one I think.

Let's see if at least another developer review the PR before merge it

**Comment by Helias** [comment]
@YggdrasilWotLK, why did you close the PR?

**Comment by YggdrasilWotLK** [comment]
Deleted the branch on accident, will open a new PR

**Pull request closed without merging**
